packages/mrml-python/src/lib.rs

```diff
@@ -79,17 +79,6 @@ impl ParserIncludeLoaderOptions {
     }
 }
 
-impl IntoPy<PyObject> for ParserIncludeLoaderOptions {
-    fn into_py(self, py: Python<'_>) -> PyObject {
-        match self {
-            Self::Noop(inner) => inner.into_py(py),
-            Self::Memory(inner) => inner.into_py(py),
-            Self::Local(inner) => inner.into_py(py),
-            Self::Http(inner) => inner.into_py(py),
-        }
-    }
-}
-
```

Comment:
The new version of pyo3 seems to take care of this now. Curious if this should still be included.